### PR TITLE
Update changelog and podspec for v2.0.0

### PR DESCRIPTION
**Pull request by corycaywood**
# Description
Update changelog and podspec for v2.0.0

# Checklist
- [x] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `fastlane ci` without errors

### CODE REVIEW

**Review by Climbatize — APPROVED** [review]
LGTM. There is also the PR that requires developers to declare environment variable instead of modifying the plist file

**Review by khairul-alam-licon — APPROVED** [review]
👍

**Comment by corycaywood** [comment]
> LGTM. There is also the PR that requires developers to declare environment variable instead of modifying the plist file

This is for the URL, App ID, and subscription key in the Demo App, right? I think that's only relevant to the SDK developers and not users of the SDK, so we shouldn't include it in the change log.

However, it would be good to have this information in the root [readme](https://github.com/rakutentech/ios-miniapp/blob/master/README.md): maybe a "How to build the Demo App" section. The Android readme has this section, but it currently just says `We are still working on this...`

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
+- **Feature:** Mini App is now downloaded as a ZIP archive and extracted. This should improve the initial launch time on a Mini App with many files.
+- **Feature:** Add support for [`navigator.geolocation.getCurrentPosition`](https://developer.mozilla.org/en-US/docs/Web/API/Geolocation/getCurrentPosition) from JavaScript. Note that the other `geolocation` APIs (`gelocation.watchPosition` and `geolocation.clearWatch`) are currently not supported.
+- **Feature:** Enable localizable strings to be overriden by the Host App. This currently applies only to dialog button text.
+- **Fixed:** Prevent cache poisoning vulnerability by verifying cached Mini App files using a SHA512 hash.
+- **Deprecated:** `MiniApp#create(appInfo:completionHandler:messageInterface)`. Your App should instead use `MiniApp#create(appId:completionHandler:messageInterface)`.
+- **Removed:** `MiniApp#create(appInfo:completionHandler:)`. Your App should instead use `MiniApp#create(appId:completionHandler:messageInterface)`.
+- **Removed:** Runtime config option for the User Agent string. This can now be set only by using the `RMAHostAppUserAgentInfo` setting in your '.plist' file.
 
+**Sample App**
+- No changes
```
Comment (nitish-a-jain) [inline CHANGELOG.md:16]: Are there no modifications in sample app for demonstrating the SDK feature additions?

**Review by nitish-a-jain — APPROVED** [review]
LGTM 👍 
I've left a comment.

File: CHANGELOG.md (hunk 1 above)
Comment (corycaywood) [reply CHANGELOG.md:16]: I don't think so. There's no changes that are observable to the users. Is this right @leorjoseph @Climbatize ?

Comment (ts-leojoseph-r) [reply CHANGELOG.md:16]: That's right. We don't have any changes in Sample app this time.

**Review by rleojoseph — APPROVED** [review]
Looks good to me :+1:

**Pull request merged**